tests/src/main/java/net/neoforged/neoforge/debug/client/ClientEventTests.java

```diff
@@ -31,4 +38,20 @@ static void playerClientChatEvent(final ClientChatEvent event, final DynamicTest
     static void clientPlayerChangeGameTypeEvent(final ClientPlayerChangeGameTypeEvent event, final DynamicTest test) {
         test.requestConfirmation(Minecraft.getInstance().player, Component.literal("Did you just change your game mode from " + event.getCurrentGameType() + " to " + event.getNewGameType() + "?"));
     }
+
+    @TestHolder(description = { "Tests if the RegisterRenderBuffersEvent event is fired and whether the registered render buffer is represented within a fixed render buffer map" }, enabledByDefault = true)
+    static void registerRenderBuffersEvent(final DynamicTest test) {
+        test.framework().modEventBus().addListener((final RegisterRenderBuffersEvent event) -> {
```

Comment:
This should use `test.eventListeners().mod()` which will only enable the event while the test is running.

Reply:
This is actually how I did it in the first place as it was mentioned before in the review, but it doesn't work that way for some reason. Breakpoints that I set inside those listeners was never reached, so I did it using test.framework().modEventBus() and it fires fine now

Reply:
Right, they are needed on startup.

Reply:
> This should use `test.eventListeners().mod()` which will only enable the event while the test is running.

That's not possible, registration events are fired at startup, there's no test enabled then.